### PR TITLE
PSI paper

### PR DESCRIPTION
**Pull request by freepengui**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.
   文章提出了一个新的 PSI 协议，其底层是一个新的并且高效的可供双方对各自输入进行秘密比较的协议。本文提出的 PSI 协议分为线上和线下两个阶段，通过将非常昂贵的密码学操作放在线下阶段进行，其线上阶段对每一个元素仅需要4个快速的域操作，从而使得其线上阶段的运行非常高效

### CODE REVIEW

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
 
 ## OT-based PSI
 
+- Faster Secure Comparisons with Offline Phase for Efficient Private Set Intersection
+  Florian Kerschbaum, Erik-Oliver Blass, Rasoul Akhavan Mahdavi
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:49]: Same for this line

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]
LGTM

File: docs/awesome-pets/papers/applications/set/psi.md (hunk 1 above)
Comment (freepengui) [reply docs/awesome-pets/papers/applications/set/psi.md]: is this OK?

**Review by Jamie-Cui — APPROVED** [review]
LGTM

**Pull request merged**